resource.c

```diff
@@ -217,6 +234,7 @@ int res_read_blk(res_blk_t *res, int pid, int flags)
 {
 	int ismeminforeq = 0;
 	int isnetdevreq = 0;
+	int isprocreq = 0;
```

Comment:
Hm, just realizing this now. Wouldn't it be cleaner to use proper booleans for this?

Reply:
agree.